### PR TITLE
[luci] Change method for getting shape/dtype in luci validation

### PR DESCRIPTION
**Pull request by llFreetimell**
Parent Issue : #4796

This commit will change method for getting shape and dtype in `luci` validation.
More specifically, `loco::shape_get` and `loco::dtype_get` to shape and dtype of circle node.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
Title `Remove loco dependency` is a bit strange: _luci_ inherits from _loco_.  Can you please be more specific?

File: compiler/luci/service/src/Validate.cpp [hunk 1]
```diff
@@ -80,23 +93,28 @@ bool validate_shape_dtype(loco::Graph *g)
...
+    assert(circle_node->shape_status() != luci::ShapeStatus::UNDEFINED);
 
     // check if output node shape is same as graph output shape
-    auto co_tensor_shape = loco::shape_get(circle_node).as<loco::TensorShape>();
     auto go_tensor_shape = graph_out->shape();
     assert(go_tensor_shape);
-    if (!(co_tensor_shape == *go_tensor_shape))
+
+    bool is_shape_valid = (circle_node->rank() == go_tensor_shape->rank());
+    for (uint32_t i = 0; is_shape_valid && i < circle_node->rank(); ++i)
```
Comment (mhs4670go) [inline compiler/luci/service/src/Validate.cpp:103]: :+1:

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**